app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/SessionsRepository.java

```diff
@@ -109,4 +111,8 @@ private void refreshCache(List<Session> sessions) {
     public void setIdDirty(boolean isDirty) {
         this.isDirty = isDirty;
     }
+
+    public Single<Response<Void>> submitSessionFeedback(SessionFeedback sessionFeedback) {
```

Comment:
I think it's better to create `SessionFeedbackRepository` class which have `submit()` method.